docs/open-source-tizen/tizen-studio/package-file-desc-guide.md

```diff
@@ -64,7 +64,7 @@ Sections are separated by 2 newline characters (`\n \n`), and each section conta
   ```
 
 
-### `build.<OS>` File
+### build.<OS> File
```

Comment:
`build.<OS>` seems like the name of file. File name has code tag like below:
`build.<OS>` File

This is one of rules on styleguide.

If you want to delete code tag (e.g. `code`), you should use `&lt;` or `&gt;` instead of `<` or `>`.